### ISSUE TITLE
Symbol of GLAP

### ISSUE THREAD

**Issue by davipirata** [calpirateparty/GLAP#1]
Do we have an official symbol? If so, we should deposit here somewhere and close the issue.

If not, we should make and vote one.

**Comment by 2041876935** [comment]
I'm okay with the LA one, though perhaps we shouldn't use the Dodger font? To be honest, I guess I'm indifferent...

![15895699_269529933465374_3755330261486266173_o](https://cloud.githubusercontent.com/assets/27784302/25602643/7235a9c4-2eaa-11e7-98d1-4b4884b7db19.jpg)

![lymcfgj-](https://cloud.githubusercontent.com/assets/27784302/25602647/7924e8b2-2eaa-11e7-9941-5faad4c09b3f.jpg)

**Comment by sptankard** [comment]
Yeah can we just roll a version with a libre font and call it a done?

On May 1, 2017 8:12:23 PM PDT, 2041876935 <notifications@github.com> wrote:
>I'm okay with the LA one, though perhaps we shouldn't use the Dodger
>font? To be honest, I guess I'm indifferent...
>
>![15895699_269529933465374_3755330261486266173_o](https://cloud.githubusercontent.com/assets/27784302/25602643/7235a9c4-2eaa-11e7-98d1-4b4884b7db19.jpg)
>
>![lymcfgj-](https://cloud.githubusercontent.com/assets/27784302/25602647/7924e8b2-2eaa-11e7-9941-5faad4c09b3f.jpg)
>
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/calpirateparty/GLAP/issues/1#issuecomment-298487236

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
